backend/internal/common/jw_api_endpoints.go

```diff
@@ -0,0 +1,9 @@
+package common
```

Comment:
Maybe we can just couple the JW API endpoints to our main env secrets file. I think it's better to have them in a dotenv anyway long term.

Maybe we could even just create a general 3rd party API endpoint structure and put it in there? JwAPIEndpoints seems a bit specific

Reply:
Yeah I agree

Reply:
done in https://github.com/cm-core-team/cm-core/pull/76/commits/ae4fd8e8fe2421765a7a6dc34678fe745c541843